src/activities/hangman/hangman.js

```diff
@@ -136,6 +136,9 @@ function processKeyPress(text) {
     // Add the character to the already typed characters
     alreadyTypedLetters.push(text);
 
+    // add
+    items.guessedText.text=form()
```

Comment:
form is not a good name for a function, maybe you can call it more explicitely.
Also, can you add a space before and after the '=' sign?